src/D2L.Security.OAuth2/Keys/Default/EcDsaJsonWebKey.cs

```diff
@@ -91,9 +82,9 @@ string y
 				validFrom: DateTime.UtcNow,
 				validTo: ExpiresAt ?? DateTime.UtcNow + Constants.REMOTE_KEY_MAX_LIFETIME,
 				keyFactory: () => {
-					var cng = BuildEcDsaCng();
-					var key = new EcDsaSecurityKey( cng );
-					return new Tuple<AsymmetricSecurityKey, IDisposable>( key, cng );
+					var publicBlob = ECCPublicKeyBlobFormatter.Instance.BuildECCPublicBlob( this );
+					var key = new ECDsaSecurityKey( publicBlob, CngKeyBlobFormat.EccPublicBlob );

```

Comment:
They undid this, since Cng isn't cross-platform, so they're trying to remove dependence on it